draft-ietf-quic-multipath.md

```diff
@@ -16,7 +16,9 @@ pi: [toc, sortrefs, symrefs]
 author:
 -
   ins: Y. Liu
```

Comment:
```suggestion
```

If you remove this line, the author names at the top of the document will properly render